common:
    lid_topic :  "/hesai/pandar"
    imu_topic :  "/imu/data_raw"
    gnss_topic:  "/gps/gps"
    time_sync_en: false         # ONLY turn on when external time synchronization is really not possible
    time_offset_lidar_to_imu: 0.0 # Time offset between lidar and IMU 
                                  # This param will take effect no matter what time_sync_en is. So if the time offset is not known exactly, please set as 0.0
preprocess:
    lidar_type: 0                # 0 for hesai, # 1 for Velodyne, 2 for ouster,
    point_step : 1               # i+= point_step  
    min_dist: .5 
    max_dist: 100

mapping:
    acc_cov:  0.1
    gyr_cov:  0.1
    b_acc_cov:  0.0001
    b_gyr_cov:  0.0001
    extrinsic_est_en: false      # true: enable the online estimation of IMU-LiDAR extrinsic
    
    GNSS_IMU_calibration_distance: 30 
    #transform the point cloud to imu frame,  P_lidar seen from IMU will be P_imu = R*P_lidar+T
    extrinsic_T: [ 0.009,  0.13,  0.09 ] 
    extrinsic_R: [ 0, 1, 0,
                  -1, 0, 0,
                   0, 0, 1]

als: 
    closest_N_files: 4
    downsample: false #true 
    als_path: "/media/eugeniu/T7/NLS_las/"   # sparse ALS from NlS

publish:
    scan_publish_en:  true       
    dense_publish_en: false      
    